example/ios/KlaviyoReactNativeSdkExample.xcodeproj/project.pbxproj

```diff
@@ -546,7 +546,7 @@
 					"$(inherited)",
 				);
 				INFOPLIST_FILE = KlaviyoReactNativeSdkExampleTests/Info.plist;
-				IPHONEOS_DEPLOYMENT_TARGET = 13.4;
```

Comment:
RN 0.77 bumped min ios support to 15.1. In some targets, I think we were missing any specifier, or had defaulted to higher than necessary

Reply:
I think same question as android, if someone where to update to these RN version they are letting go of lower iOS versions so it's not really us who is responsible for the higher min deployment target?

Reply:
I want to confirm this, but I believe this is just dependent on the host app's version of RN. Since the example is now using 0.77, the min os version expectation has increased, but an RN app using 0.74 for example should still support 13.